src/scripts/sync_synonym.py

```diff
@@ -0,0 +1,66 @@
+"""Whenever new synonym or label in upstream source, want to add it to Mondo. If the label is not the Mondo label or
+     existing synonym, it will become a synonym.
+"""
+import os
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import Union
+
+import pandas as pd
+
+# TODO: import via config
+# Vars
+SCRIPTS_DIR = os.path.dirname(os.path.realpath(__file__))
+PROJECT_DIR = os.path.join(SCRIPTS_DIR, '..', '..')
+ONTOLOGY_DIR = os.path.join(PROJECT_DIR, 'src', 'ontology')
+# REPORTS_DIR = os.path.join(ONTOLOGY_DIR, 'reports')
+# METADATA_DIR = os.path.join(ONTOLOGY_DIR, 'metadata')
+# CONFIG_DIR = os.path.join(ONTOLOGY_DIR, 'config')
+SYNC_DIR = os.path.join(ONTOLOGY_DIR, 'sync')
+TEMP_DIR = os.path.join(ONTOLOGY_DIR, 'tmp')
+CACHE_DIR = TEMP_DIR
+
+
+
+# TODO
+# TODO: If synonym is not already in Mondo, add it into a sync/%-synonyms.robot.template.tsv
```

Comment:
Keep in mind the synonym scope and synonym type and xrefs for the synonym. 

For example, for MONDO:0020724, cerebral cavernous malformation 1 here are a few synonyms:
"cavernous malformations of CNS and retina" RELATED [OMIM:116860]
"CCM" RELATED ABBREVIATION [MONDO:Lexical, OMIM:116860]

The Mondo documentation has more information on these: https://mondo.readthedocs.io/en/stable/editors-guide/f-entities/#synonyms

Reply:
Ahh, interesting. I didn't know about this stuff, thanks. I updated your comment to put in some sub-tasks, etc.

Reply:
@twhetzel @matentzn I'm only supposed to query synonym types (DEPRECATED, MISSPELLING, ABBREVIATION, etc) from Mondo, right? Our sources don't mark synonyms with any of these kinds of annotations, right? (I double checked a couple sources and didn't see any of that yet).